tests/e2e/test_amq_streamer_creation.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, workloads
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.amq import AMQ
+from ocs_ci.ocs.exceptions import (ResourceWrongStatusException)
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def test_fixture_amq(request, storageclass_factory):
+    cephfs_sc_obj = storageclass_factory(interface=constants.CEPHFILESYSTEM, sc_name='amq-workload')
```

Comment:
can we use `rook-ceph-cephfs` sc instead? We should recommend users to use the storage class OCS creates.

Reply:
@SvetlanaRedHat this is a good point, please file github issue and you can work on this as follow on PR.

Reply:
@keesturam @vasukulkarni i had a word with @SvetlanaRedHat  about this. I have not any objection to use CephFS being used for AMQ Streams (Kafka) however, in our team we are investigating to use Ceph Block Storage RBD for Kafka workload. And would be recommending this to customers. So we should definitely add RBD storage class to QA tests. I am open for a longer conversation on this.

Reply:
@ksingh7 what was the access mode used in your testing? my understanding is amq needs storage access from multiple nodes (rwx), which was the primary driving factor to use cephfs.